docker/README.md

```diff
@@ -305,3 +305,39 @@ You’ll need to install the [php-debug](https://atom.io/packages/php-debug) pac
 1. You should be able to set breakpoints now:
 
 	![Screen animation showing setting a breakpoint](https://user-images.githubusercontent.com/746152/37093212-591fe7d8-21ed-11e8-8352-47839ce58964.gif)
+
+#### Remote debugging with PhpStorm editor
+
+Below are instructions for starting a debug session in PhpStorm that will listen to activity on your Jetpack docker.
+If you prefer a visual aid, here is a quick screen cast.
+
+1. Configure your browser extension to use 'JETPACK' for its session ID.
```

Comment:
Why don't recommending just `PHPSTORM` ?

Reply:
yes lets keep PHPSTORM

Reply:
Because I'm loyal to the Jetpack brand. 😛 
j/k `PHPSTORM` is fine by me.